plugins/content/view_history/lib/model/filters/kViewHistoryUserEntryAdvancedFilter.php

```diff
@@ -68,11 +68,21 @@ public function applyCondition(IKalturaDbQuery $query)
 		$limit = $query->getLimit() ? $query->getLimit() : ($this->filterLimit ? $this->filterLimit : self::ENTRIES_COUNT);
 		
 		$query->setLimit(self::ENTRIES_COUNT);
+		if ($this->filterLimit)
+		{
+			KalturaLog::info ("Overriding filter limit to " . self::ENTRIES_COUNT);
+			$this->overrideFilterLimit = self::ENTRIES_COUNT;
```

Comment:
$this->overrideFilterLimit   will be override in line 84 , what was the purpose of this line?